astroquery/ipac/irsa/tests/test_irsa.py

```diff
@@ -5,22 +5,23 @@
 import numpy as np
 
 import pytest
+from astropy.coordinates import SkyCoord
 from astropy.table import Table
-import astropy.coordinates as coord
 import astropy.units as u
 
 from astroquery.utils.mocks import MockResponse
-from astroquery.utils import commons
 from astroquery.ipac.irsa import Irsa, conf
 from astroquery.ipac import irsa
 
 DATA_FILES = {'Cone': 'Cone.xml',
               'Box': 'Box.xml',
               'Polygon': 'Polygon.xml'}
 
-OBJ_LIST = ["m31", "00h42m44.330s +41d16m07.50s",
-            commons.GalacticCoordGenerator(l=121.1743, b=-21.5733,
-                                           unit=(u.deg, u.deg))]
+OBJ_LIST = [
+    "m31",
+    "00h42m44.330s +41d16m07.50s",
+    SkyCoord(l=121.1743 * u.deg, b=-21.5733 * u.deg, frame="galactic")
```

Comment:
keep only the PR relevant changes, e.g. getting rid of the coorgenerator, please don't reformat